src/main/kotlin/no/nav/dagpenger/journalføring/skanning/JournalføringSkanning.kt

```diff
@@ -97,17 +106,32 @@ class JournalføringSkanning(val env: Environment) :
         return behov
     }
 
-    private fun containsJsonDokument(key: String, behov: Behov): Boolean {
+    private fun registerMetrics(behov: Behov) {
+        val rettighetstype = when {
+            behov.hasSøknadRettighetsType() -> behov.getHenvendelsesType().getSøknad().getRettighetsType().toString()
+            behov.hasEttersendingRettighetsType() -> behov.getHenvendelsesType().getEttersending().getRettighetsType().toString()
+            else -> "N/A"
```

Comment:
not sure if `N/A` is a legal prometheus name.
 `unknown` or `not_set` ?

Reply:
Good catch, I'll change it to "unknown"